<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>criarContDialog</class>
 <widget class="QDialog" name="criarContDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Criar Conteúdo</string>
  </property>
  <widget class="QLineEdit" name="tituloLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Digite o Título</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>46</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Título:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="procurarImgBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Procurar Imagem no Computador</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imgLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Imagem:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vídeo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="procurarVideoBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Procurar vídeo no computador</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="criarBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Criar a conteúdo</string>
   </property>
   <property name="text">
    <string>Criar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Cancelar e sair</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="escolheComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="escolheLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Escolha a Categ.:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="contProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="criandoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criando conteúdo:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
